dist/expected.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package dist
+
+import "github.com/gonum/quad"
+
+// ExpectedFixed numerically computes the expected value of the input function
+// under the probability distribution specified by the Quantiler. That
+// is, ExpectedFixed computes
+//
+//  ev ≈ E[f] = int_-inf^inf f(x) p(x) dx
+//
+// using the number of quadrature locations specified by evals. Concurrent
+// sets the number of allowed concurrent evaluations of f.
+//
+// ExpectedFixed is a wrapper around quad.Fixed. See the documentation for more
+// information.
+func ExpectedFixed(f func(float64) float64, q Quantiler, evals, concurrent int) (ev float64) {
```

Comment:
Yes I am being very picky, and probably too picky.  I'd rather say something about an issue (that is hopefully not completely stupid) and have it get shot down than not mention it at all.  I think there might be some middle grounds where the result is the same with same concurrent input, but it isn't obvious how much a performance hit there would be.  Anyway, as long as you're ok with it then I'm fine, although you might want a note that mentions that the output won't be exactly equal if concurrent >= 2.